index.rst

```diff
@@ -145,16 +154,24 @@ An example report generated by Robot Framework is available `here <_static/tts_c
 
 .. _Robot Framework: https://robotframework.org
 
+One of the key aspects of requirement verification is to be able to map a particular test with a particular requirement it verifies.
+For that, Robot framework allows us to add tags to test reports.
+Using those tags we can assign the list of requirements that are verified by each individual test.
+
 .. _activity-sequence-diagrams:
 
 Activity Sequence Diagrams
 --------------------------
 
+Most of the software requirement verification we perform consist of demonstrating that a particular component executes a particular set of actions.
```

Comment:
consists
you also have two particulars :) you could change the latter to "specific"